internal/cmd/apit/setsnippet.go

```diff
@@ -0,0 +1,92 @@
+package apit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/saucelabs/saucectl/internal/apitest"
+	"github.com/saucelabs/saucectl/internal/http"
+	"github.com/spf13/cobra"
+)
+
+func SetSnippetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-snippet NAME FILE_NAME|- [--project PROJECT_NAME]",
+		Short: "Set a vault snippet",
+		Long: `
+Set/update a snippet in a project's vault. If a snippet NAME is already in the vault,
+the value will be updated otherwise a new snippet will be added. You can set a snippet's
+value by providing a path to a file defining the snippet or use "-" to read from stdin.
+
+Use [--project] to specify a project by its name or run without [--project] to choose from
+a list of projects
+`,
```

Comment:
I think it'd be nice to also add an `Example:` (it's a [dedicated field](https://github.com/saucelabs/saucectl/blob/3fa3f30b790b226948af312a60d37271acb6334c/internal/cmd/run/xcuitest.go#L42-L42) in cobra). At least one with stdin and one with filename?

Considering that our current API testing solution is targeting QAs and not developers, examples are probably very welcome 💭